src/components/AddItem.js

```diff
@@ -1,11 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { db } from '../lib/firebase';
+
+const initialFormState = {
+  itemName: '',
+  timeFrame: 7,
+  lastPurchased: null,
+};
 
 const AddItem = () => {
+  const token = 'user token';
+  const history = useHistory();
+  const [formData, setFormData] = useState(initialFormState);
+  const [error, setError] = useState('');
+  console.log(error);
+
+  const handleFormChange = (event) => {
+    event.preventDefault();
+    setFormData({
+      ...formData,
+      [event.target.name]: event.target.value,
+    });
+  };
```

Comment:
Just a quick comment...I think. you may need to `parseInt()` the `timeFrame` value before sending it. If I remember correctly it's supposed to be saved as an int, and right now it's getting saved as a string, and I think even if you set it's value on the radio to an int, it will still send as a string...need to convert it to an `int`.

Reply:
> Just a quick comment...I think. you may need to `parseInt()` the `timeFrame` value before sending it. If I remember correctly it's supposed to be saved as an int, and right now it's getting saved as a string, and I think even if you set it's value on the radio to an int, it will still send as a string...need to convert it to an `int`.

Would it be sent in as a number if the value were set like this:
```
<input
            type="radio"
            id="soon"
            name="timeFrame"
            defaultChecked
            value={7}
            onChange={handleFormChange}
          />
```
instead of `value="7"`

Reply:
So I tried the above changes and the value for timeframe is now a number so I think that works, but if parseInt is better we can change it

Reply:
I'm not 100% sure @DeeDowns...would be good to test it out, then it will be a simpler change, for sure.

Reply:
Oh perfect...no need for `parseInt` then.

Reply:
Nice! That's a clean solution!

Reply:
We did end up having to use parseInt to pass it as a number to the db.